src/utils/formatTags.test.ts

```diff
@@ -0,0 +1,21 @@
+import { formatTags } from "./formatTags";
+
+const tags = [
+  '{"React","Typescript","Javascript","Front-end","Back-end","CSS","HTML","SQL"}',
+];
+
+const empty = ["{}"];
```

Comment:
emptyTags or emptyObj , needs a friend on the end of the variable name to clarify